zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/signal/SignalBroadcastProcessor.java

```diff
@@ -173,14 +173,21 @@ private void activateElement(
   @Override
   public ProcessingError tryHandleError(
       final TypedRecord<SignalRecord> command, final Throwable error) {
-    if (error instanceof final ForbiddenException exception) {
-      rejectionWriter.appendRejection(
-          command, exception.getRejectionType(), exception.getMessage());
-      responseWriter.writeRejectionOnCommand(
-          command, exception.getRejectionType(), exception.getMessage());
-      return ProcessingError.EXPECTED_ERROR;
-    }
 
-    return ProcessingError.UNEXPECTED_ERROR;
+    switch (error) {
+      case final NotFoundException exception:
```

Comment:
❓ Is this a breaking change ?

Reply:
I removed it in a later commit as we don't want any not founds in signal broadcasting anyway. This slipped in at some point during the rearchitecture but it's wrong.